python/llm/src/bigdl/llm/transformers/qlora.py

```diff
@@ -234,8 +230,7 @@ def _create_new_module(create_new_module_func, lora_config, adapter_name, target
 
 @dataclass
 class LoraConfig(LoraConfigBase):
-    lora: bool = field(default=False, metadata={"help": "enable lora"})
-    qa_lora: bool = field(default=False, metadata={"help": "enable qa-lora"})
+    training_mode: str = field(default="qlora", metadata={"help": "determine training mode"})
```

Comment:
add more info in help: what are possible values, what is the default value

Reply:
Done.